src/flp/types.rs

```diff
@@ -419,55 +419,60 @@ impl<F: FftFriendlyFieldElement> Type for Histogram<F> {
     }
 }
 
-/// A sequence of counters. This type uses a neat trick from [[BBCG+19], Corollary 4.9] to reduce
-/// the proof size to roughly the square root of the input size.
+/// A sequence of integers in range `[0, 2^bits)`. This type uses a neat trick from [[BBCG+19],
+/// Corollary 4.9] to reduce the proof size to roughly the square root of the input size.
 ///
 /// [BBCG+19]: https://eprint.iacr.org/2019/188
 #[derive(Debug, PartialEq, Eq)]
-pub struct CountVec<F, S> {
+pub struct SumVec<F, S> {
```

Comment:
Renaming this changes the crate's public API. This isn't a problem for VDAF compliance, since no draft specifies either `CountVec` or `SumVec`, but it does mean we'd want to bump the crate version before releasing a version that includes this. Unless, that is, we can put back in a `SumVec` type that's just an alias for `CountVec` with `bits = 1`, and then the API won't change. I'm comfortable with either.

Reply:
We could just define an alias, yeah. The only problem I foresee is that `CountVec::new()` would to `SumVec::new()`, which is ambiguous because this type doesn't take in a `bits` parameter. One option is to do

```rust
pub struct CountVec(SumVec);
```

instead of

```rust
pub type CountVec = SumVec
```

Reply:
If we're bumping the version, it might be a good idea to merge this into a branch other than main so that we can accumulate more API-breaking changes before bumping.

Reply:
For what it's worth, we do have some breaking changes floating on main currently -- the output share/aggregate share `Encode`/`ParameterizedDecode` trait bounds, and moving some items behind the experimental feature.

Reply:
I'm good either way -- just let me know what y'all decide :)

Reply:
Increase the major version!

Reply:
Oh, thanks @divergentdave, I forgot we had such changes on `main` already. Plus we know there will be API breaking stuff for VDAF-04.

Reply:
Sweet!